charmstoretesting/server.go

```diff
@@ -59,7 +73,17 @@ func (s *Server) Close() {
 	s.srv.Close()
 }
 
+func (s *Server) client() *csclient.Client {
```

Comment:
I wonder if it might be better to have:

    // Client returns a new client that  will talk to the Server using basic
    // (non-macaroon) authentication.
    func (s *Server) NewClient() *csclient.Client

so that it's easy for tests to do this too.

I think you probably want:

    func (s *Server) URL() string

so that tests can also create their own csclient.Client
easily with whatever params they want.